src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -4,27 +4,151 @@
 #include "pch.h"
 #include "ConptyConnection.h"
 
-#include <Windows.h>
+#include <windows.h>
+#include <sstream>
```

Comment:
`sstream` should be in `LibraryIncludes.h`

Reply:
almost certainly.